scss/civicrm/common/_modals.scss

```diff
@@ -163,4 +163,8 @@
   .dataTables_wrapper {
     box-shadow: none;
   }
+
+  .crm-form-block h3 {
```

Comment:
Do you think we should nest it ?
```
.crm-form-block {
  h3 {
  }
}
```

Reply:
No, why? If there is no other elements, why make it nested?